gammapy/estimators/flux.py

```diff
@@ -42,12 +42,12 @@ class FluxEstimator(Estimator):
         Re-optimize other free model parameters.
     selection_optional : list of str
         Which additional quantities to estimate. Available options are:
-
+            * "all": all the optional steps are executed
             * "errn-errp": estimate asymmetric errors.
             * "ul": estimate upper limits.
             * "norm-scan": estimate fit statistic profiles.
```

Comment:
It should be `scan` instead of `norm-scan`, no?

Reply:
yes, fixed in the latest commit